inc/Widget/SmailyWidget.php

```diff
@@ -105,12 +105,70 @@ public function widget( $args, $instance ) {
 
 			echo '
 				<div class="smaily-newsletter-alert">
-				<p>' . esc_html( $message ) . '
-				<span class="smaily-newsletter-closebtn" onclick="this.parentElement.style.display=\'none\'">&times;</span>
-				</p>
+					<p>' . esc_html( $message ) . '
+						<span class="smaily-newsletter-closebtn" onclick="this.parentElement.style.display=\'none\'">&times;</span>
+					</p>
 				</div>
 				';
 		}
+
+		// Render the style property only if it has a value.
+		$smaily_button_style_elements = [];
+		if ( ! empty( $instance['submit_button_text_color'] ) ) {
+			$smaily_button_style_elements[] = 'color: ' . esc_attr( $instance['submit_button_text_color'] );
+		}
+
+		if ( ! empty( $instance['submit_button_color'] ) ) {
+			$smaily_button_style_elements[] = 'background-color: ' . esc_attr( $instance['submit_button_color'] );
+		}
+		// Create and return a new string by concatenating all the elements in an array.
+		$smaily_button_style = join(' ; ',$smaily_button_style_elements);
+
+		// Set default layout option.
+		$allowed_layouts = array( 'layout-1', 'layout-2', 'layout-3', 'layout-4', 'layout-5' );
+		$selected_layout = isset( $instance['form_layout'] ) ? $instance['form_layout'] : 'layout-1';
+		$selected_layout = in_array( $selected_layout, $allowed_layouts, true ) ? $selected_layout : 'layout-1';
+		// Layout options to echo.
+		switch ( $selected_layout ) {
+			// Layout 1.
+			case 'layout-1':
+				$layout = '<div class="widget_smaily_widget smaily-layout-container smaily-layout-1">
```

Comment:
Class `widget_smaily_widget` is already provided by the widget and is not necessary here. Besides it does not have any effect, because `div.smaily-layout-container` CSS rules is required to be nested inside `widget_smaily_widget`.